solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudHttp2SolrClient.java

```diff
@@ -34,7 +34,7 @@
  * @since solr 8.0
  */
 @SuppressWarnings("serial")
```

Comment:
Should CloudHttp2SolrClient be deprecated and ask users to use CloudSolrClient? Hmm, that sounds confusing for those who moved from CloudSolrClient to CloudHttp2SolrClient in 8.x :) So perhaps a point to wait closer to 10.0 to explain this dance to users..

Reply:
Yeah; makes sense to wait.